trainGeneticAlgorithm.py

```diff
@@ -44,7 +46,7 @@ def main():
 	
 	pygame.font.init()
 
-	while game.play:
+	while game.play and game.num_generations < max_generations:
```

Comment:
Just to be sure - is `game.num_generations` zero indexed?

Reply:
Yes